src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    //to store the sentence
+    private static String line;
+    //to keep track of the number of task
+    private static int count = 0;
+    private static Scanner in = new Scanner(System.in);
+    //Array of object
+    private static Task[] tasks = new Task[100];
+
+
+    public static void main(String[] args) throws DukeException {
+
+        System.out.println("Hello! I'm Duke\nWhat can i do for you?\n");
+
+        while(true){
+
+            line = in.nextLine();
+            try {
+                if(!(line.contains("bye")||line.contains("todo")||line.contains("deadline")||line.contains("event")
+                        ||line.contains("list")||line.contains("done"))){
+                    throw new DukeException();
+                }
+                if (line.equals("bye")) {
+                    System.out.println("Bye. Hope to see you again soon!\n");
+                    break;
+                } else if (line.contains("todo")) {
+                    try {
+                        if (line.length() == 4) {
```

Comment:
It would be better if you could add more comments to explain why the exception is thrown if the length of the line equals 4.